app/main.py

```diff
@@ -1,8 +1,64 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        if 1 <= comfort_class <= 7:
+            self.comfort_class = comfort_class
+        else:
+            raise ValueError("comfort_class should be between 1 and 7")
+        if 1 <= clean_mark <= 10:
+            self.clean_mark = clean_mark
+        else:
+            raise ValueError("clean_mark should be between 1 and 10")
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        if 1.0 <= distance_from_city_center <= 10.0:
+            self.distance_from_city_center = distance_from_city_center
+        else:
+            raise ValueError(
+                "distance_from_city_center should be between 1.0 and 10.0"
+            )
+        self.clean_power = clean_power
+        if 1.0 <= average_rating <= 5.0:
+            self.average_rating = round(average_rating, 1)
+        else:
+            raise ValueError("average_rating should be between 1.0 and 5.0")
```

Comment:
try replacing the duplicate code

Reply:
I feel like there is another way to test the arguments, but I don't know what